yt_dlp/extractor/mojevideo.py

```diff
@@ -0,0 +1,115 @@
+            'display_id': 'chlapci_dobetonovali_sme_mame_hotovo',
+            'description': 'md5:a0822126044050d304a9ef58c92ddb34',
+            'thumbnail': 'https://fs5.mojevideo.sk/imgfb/250236.jpg',
+            'duration': 21.0,
+            'upload_date': '20230919',
+            'timestamp': 1695129706,
+            'like_count': int,
+            'dislike_count': int,
+            'view_count': int,
+            'comment_count': int,
+        },
+    }, {
+        # 720p
+        'url': 'https://www.mojevideo.sk/video/14677/den_blbec.html',
+        'md5': '517c3e111c53a67d10b429c1f344ba2f',
+        'info_dict': {
+            'id': '83575',
+            'ext': 'mp4',
+            'title': 'Deň blbec?',
+            'display_id': 'den_blbec',
+            'description': 'I maličkosť vám môže zmeniť celý deň. Nikdy nezahadzujte žuvačky na zem!',
+            'thumbnail': 'https://fs5.mojevideo.sk/imgfb/83575.jpg',
+            'duration': 100.0,
+            'upload_date': '20120515',
+            'timestamp': 1337076481,
+            'like_count': int,
+            'dislike_count': int,
+            'view_count': int,
+            'comment_count': int,
+        },
+    }, {
+        # 1080p
+        'url': 'https://www.mojevideo.sk/video/2feb2/band_maid_onset_(instrumental)_live_zepp_tokyo_(full_hd).html',
+        'md5': '64599a23d3ac31cf2fe069e4353d8162',
+        'info_dict': {
+            'id': '196274',
+            'ext': 'mp4',
+            'title': 'BAND-MAID - onset (Instrumental) Live - Zepp Tokyo (Full HD)',
+            'display_id': 'band_maid_onset_(instrumental)_live_zepp_tokyo_(full_hd)',
+            'description': 'Výborná inštrumentálna skladba od skupiny BAND-MAID.',
+            'thumbnail': 'https://fs5.mojevideo.sk/imgfb/196274.jpg',
+            'duration': 240.0,
+            'upload_date': '20190708',
+            'timestamp': 1562576592,
+            'like_count': int,
+            'dislike_count': int,
+            'view_count': int,
+            'comment_count': int,
+        },
+    }, {
+        # 720p
+        'url': 'https://www.mojevideo.sk/video/358c8/dva_nissany_skyline_strielaju_v_londyne.html',
+        'only_matching': True,
+    }, {
+        # 720p
+        'url': 'https://www.mojevideo.sk/video/2455d/gopro_hero4_session_nova_sportova_vodotesna_kamera.html',
+        'only_matching': True,
+    }, {
+        # 1080p
+        'url': 'https://www.mojevideo.sk/video/352ee/amd_rx_6800_xt_vs_nvidia_rtx_3080_(test_v_9_hrach).html',
+        'only_matching': True,
+    }, {
+        # 1080p
+        'url': 'https://www.mojevideo.sk/video/2cbeb/trailer_z_avengers_infinity_war.html',
+        'only_matching': True,
+    }]
+
+    def _real_extract(self, url):
+        page_id, display_id = self._match_valid_url(url).groups()
+        webpage = self._download_webpage(url, page_id)
+
+        video_id = self._search_regex(r'\bvId\s*=\s*(\d+)', webpage, 'video id')
```

Comment:
This is the same as the page_id in decimal. I would use the ID in the URL as the video_id and then use this one only for the request, probably with a fallback to `str(int(video_id, 16))`

Reply:
Sharp-eyed!

Done. See 8b6ccef342820df7b0ad7434469a1921380e93da .